### ISSUE TITLE
Support multiple font-variants

### ISSUE THREAD

**Issue by finnp** [styled-components/css-to-react-native#175]
The following:
```jsx
styled.view`
 font-variant: small-caps oldstyle-nums;
`
```

 should map to
```jsx
<View style={{fontVariant: ['small-caps', 'oldstyle-nums']}}
```

It looks like instead it only chooses one of the font-variants provided.

**Comment by jacobp100** [comment]
You're right. Embarrassing too - I implemented `fontVariant` in react-native 😳😅

If you wanted to take a crack at it, replace [this](https://github.com/styled-components/css-to-react-native/blob/master/src/transforms/index.js#L42-L44) with something like [this](https://github.com/styled-components/css-to-react-native/blob/master/src/transforms/fontFamily.js#L10-L14)

**Comment by jacobp100** [comment]
Published 3.1.0
